test/functionalTest/cases/3693_covered_notifications_in_subscritpions/covered_notifications_in_subscritpions.test

```diff
@@ -0,0 +1,488 @@
+
+
+
+echo '06. POST /v2/entities, to create Room2 with temperature and pressure.'
+echo '====================================================================='
+payload='{
+  "id": "Room2",
+  "type": "Room",
+  "temperature": {
+    "value": 23,
+    "type": "Float"
+  },
+  "pressure": {
+    "value": 720,
+    "type": "Integer"
+  }
+}'
+orionCurl --url /v2/entities --payload "$payload"
+echo
+echo
+
+
+
+echo '07. POST /v2/subscriptions, to create subscription with covered=false and some attributes in attrs which is not present in entities.'
+echo '===================================================================================================================================='
+payload='{
+  "description": "A subscription to get info about Room2",
+  "subject": {
+    "entities": [
+      {
+        "id": "Room2",
+        "type": "Room"
+      }
+    ],
+    "condition": {
+      "attrs": [
+        "pressure"
+      ]
+    }
+  },
+  "notification": {
+    "http": {
+      "url": "http://localhost:'$LISTENER_PORT'/notify"
+    },
+    "attrs": [
+      "temperature",
+      "humudity",
+      "brightness"
+    ],
+    "covered": false
+  }
+}'
+orionCurl --url /v2/subscriptions --payload "$payload"
+subId=$(echo "$_responseHeaders" | grep Location | awk -F/ '{ print $4 }' | tr -d "\r\n")
+echo
+echo
+
+
+
+echo '08. GET /v2/subscriptions/subscription_id, to check the covered=false.'
+echo '======================================================================'
+orionCurl --url /v2/subscriptions/${subId}
+echo
+echo
+
+
+
+echo '09. PATCH /v2/entities/Room2/attrs, to update the value of pressure attribute.'
+echo '=============================================================================='
+payload='{
+  "temperature": {
+    "value": 24,
+    "type": "Float"
+  },
+  "pressure": {
+    "value": 730,
+    "type": "Float"
+  }
+}'
+orionCurl --url /v2/entities/Room2/attrs --payload "$payload" -X PATCH
+echo
+echo
+
+
+
+echo "10. Dump the accumulator, see the no missing attribute in the notification added."
+echo "================================================================================="
+accumulatorDump
+accumulatorReset
+echo
+echo
+
+
+
+echo '11. POST /v2/subscriptions, to create subscription with covered=foo and see error.'
+echo '=================================================================================='
+payload='{
+  "description": "A subscription to get info about Room1",
+  "subject": {
+    "entities": [
+      {
+        "id": "Room1",
+        "type": "Room"
+      }
+    ],
+    "condition": {
+      "attrs": [
+        "pressure"
+      ]
+    }
+  },
+  "notification": {
+    "http": {
+      "url": "http://localhost:'$LISTENER_PORT'/notify"
+    },
+    "attrs": [
+      "temperature",
+      "temp1",
+      "temp2"
+    ],
+    "covered": "foo"
+  }
+}'
+orionCurl --url /v2/subscriptions --payload "$payload"
+subId=$(echo "$_responseHeaders" | grep Location | awk -F/ '{ print $4 }' | tr -d "\r\n")
```

Comment:
```suggestion
```

As the expected result is error so you don't need to keep any subId.

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4068/commits/a1770e3bc47943ca47b075ec25a9dd3ed512cda5